.travis.yml

```diff
@@ -8,12 +8,13 @@
 
 dist: xenial
 language: python
+# Use pypy3.6-7.0.0 to have 1.12.0 version of CFFI which has `dlclose`
 python:
  - "2.7"
  - "pypy"
  - "3.6"
  - "3.7"
- - "pypy3.5"
+ - "pypy3.6-7.0.0"
```

Comment:
Great! Maybe we should keep an eye on pypy release and update from travis-pre-compiled virtual environment for updating this ..

Reply:
Yes. Hopefully they will add some explanation to the official documentation...